lab-gurpreet/ReadMe

```diff
@@ -0,0 +1,12 @@
+As a developer----
+
+    instruction :-
+
+       once you clone the file in your root directory. open it up in the atom.
+       you don't need special dependencies for this project.
+       you need to run command npm init in order to run node.
```

Comment:
You want to tell users to run 'npm install' instead of 'npm init'. Do you understand the difference?